src/main/app/src/Components/Landscape/Map/MapRelation/MapRelation.tsx

```diff
@@ -10,49 +13,51 @@ interface Props {
 
 /**
  * Returns a choosen Map Relation
- * @param element Choosen SVG Element from our Landscape Component
+ * TODO work with items and relation objects
  */
 const MapRelation: React.FC<Props> = ({ sourceIdentifier, targetIdentifier, type, findItem }) => {
   const sourceGroupNameStart = sourceIdentifier.indexOf('/') + 1;
   const sourceRelation = sourceIdentifier.substr(sourceGroupNameStart);
-
+  const classes = componentStyles();
   const targetGroupNameStart = targetIdentifier.indexOf('/') + 1;
   const targetRelation = targetIdentifier.substr(targetGroupNameStart);
 
   const sourceTitle = sourceIdentifier.split('/').pop();
   const targetTitle = targetIdentifier.split('/').pop();
 
+  const title = 'Relation ' + sourceTitle + ' ⇄ ' + targetTitle;
   return (
-    <div className='mapRelation'>
-      <div className='titleContainer'>
-        <span className='title'>
-          {sourceTitle} {'⇄'} {targetTitle}
-        </span>
-      </div>
-      <div className='mapRelationContent'>
-        <span className='type'>Type: {type}</span>
-        <div className='relationsContent'>
-          <span
-            className='relation'
-            key={sourceIdentifier}
-            onClick={() => {
-              findItem(sourceIdentifier);
-            }}
-          >
-            {sourceRelation}
-          </span>
-          <span
-            className='relation'
-            key={targetIdentifier}
-            onClick={() => {
-              findItem(targetIdentifier);
-            }}
-          >
-            {targetRelation}
-          </span>
-        </div>
-      </div>
-    </div>
+    <Card className={classes.card}>
+      <CardHeader title={title} className={classes.cardHeader}/>
+      <CardContent>
+        <span>Type: {type}</span>
+        <br />
+        <br />
+        Source:
+        <br />
```

Comment:
Instead of using this elements to style the text, consider using `Typography` from Material UI. I also saw it in the `Events.tsx`.